docs/source/scala/quickstart.md

```diff
@@ -0,0 +1,159 @@
+# Quick Start Guide
+In this section, we will see how we can work with the Raphtory Scala APIs to create and analyse graphs. The foundations that we have build in the introductory chapters are as much applicable to this section therefore we won't be revisiting them here again!
+
+## Installing Dependencies
+To work with a Scala project you would need Scala, Java and SBT as development dependencies. The dependencies that we have tested the project discussed in this section to work with Raphtory are Scala version `2.13.7`, Java version `11.0.11.hs-adpt` and SBT version `1.8.0`.
+
+**Note**: You can manage these dependencies using [SDKMAN](https://sdkman.io/), if you like.
```

Comment:
There is proper syntax for adding notes like so:
`````
```{note}
some note goes here
```
`````

Can you also add the sdkman commands for actually installing things here (they used to be in a file that you now deleted)